docs/layouts/partials/menu-footer.html

```diff
@@ -9,3 +9,12 @@
 </div>
 <!-- Place this tag in your head or just before your close body tag. -->
 <script async defer src="https://buttons.github.io/buttons.js"></script>
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script type="text/javascript">
+ docsearch({
+ apiKey: '656714d8c3d5f7138b76633780faf6ba',
+ indexName: 'docksal',
+ inputSelector: '#search-by',
+ debug: true // Set debug to true if you want to inspect the dropdown
+ });
+</script>
```

Comment:
newline missing

Reply:
True. And the debug needs to be set back to false. Will do.

Reply:
False is the default and I forgot that I had set it to true when trying to get the search results box to appear above the text of the documentation.